docs/master/api-reference/directives.md

```diff
@@ -2867,57 +2864,79 @@ Install the dependency [mll-lab/graphql-php-scalars](https://github.com/mll-lab/
 
     composer require mll-lab/graphql-php-scalars
 
-It contains the scalar type `Mixed`, which enables the dynamic query capabilities.
+### Usage
 
 ```graphql
-scalar Mixed @scalar(class: "MLL\\GraphQLScalars\\Mixed")
+type Person {
+    id: ID!
+    age: Int!
+    height: Int!
+    type: String!
+    hair_colour: String!
+}
+
+type Query {
+    people(
+        where: WhereConstraints @whereConstraints(columns: ["age", "type", "haircolour", "height"])
+    ): [Person!]! @all
+}
 ```
 
-Add an enum type `Operator` to your schema. Depending on your
-database, you may want to allow different internal values. This default
-should work for most databases:
+Lighthouse generates definitions for an `Enum` type and an `Input` type
+that are restricted to the defined columns.
 
 ```graphql
-enum Operator {
-    EQ @enum(value: "=")
-    NEQ @enum(value: "!=")
-    GT @enum(value: ">")
-    GTE @enum(value: ">=")
-    LT @enum(value: "<")
-    LTE @enum(value: "<=")
-    LIKE @enum(value: "LIKE")
-    NOT_LIKE @enum(value: "NOT_LIKE")
+"Dynamic WHERE constraints for the `where` argument on the query `people`.
```

Comment:
You still have the unterminated string here that'll throw an error if people copy this code into their own schema.